redhat/opm-common.spec

```diff
@@ -12,7 +12,7 @@ License:        GPL-3.0
 Group:          Development/Libraries/C and C++
 Url:            http://www.opm-project.org/
 Source0:        https://github.com/OPM/%{name}/archive/release/%{version}/%{tag}.tar.gz#/%{name}-%{version}.tar.gz
-BuildRequires:  git doxygen bc devtoolset-6-toolchain ecl-devel openmpi-devel mpich-devel zlib-devel
+BuildRequires:  git doxygen bc devtoolset-6-toolchain openmpi-devel mpich-devel zlib-devel
```

Comment:
I *think* zlib-devel can go with libecl.

Reply:
I'll try to remove it and assess the impact.

Reply:
Build goes through without `zlib-devel`, but that's possibly a false positive.  We won't really know until we try a packaging run.